scripts/check_frontend_test_coverage_test.py

```diff
@@ -354,6 +354,69 @@ def test_fully_covered_filenames_is_not_sorted(self) -> None:
                     ],
                 )
 
+    def test_check_if_file_should_be_checked(self) -> None:
+        self.assertTrue(
+            check_frontend_test_coverage.check_if_file_should_be_checked(
+                'file.ts', ['file.ts']
+            )
+        )
+        self.assertTrue(
+            check_frontend_test_coverage.check_if_file_should_be_checked(
+                'file.ts', ['/path/to/file.ts']
+            )
+        )
+        self.assertFalse(
+            check_frontend_test_coverage.check_if_file_should_be_checked(
+                'file.ts', ['anotherfile.ts']
+            )
+        )
+        self.assertFalse(
+            check_frontend_test_coverage.check_if_file_should_be_checked(
+                'file.ts', ['/path/to/anotherfile.ts']
+            )
+        )
+        self.assertFalse(
+            check_frontend_test_coverage.check_if_file_should_be_checked(
+                '/path/to/file.ts', ['anotherfile.ts']
```

Comment:
Please test edge cases too like:
* `files_to_check = None`
* `files_to_check = []`
* `file_path = ''`
* `file_path` points to a directory

Reply:
Done except directory, do we need to test that since its for files not directories (and I changed to only support file paths so in the case a directory is passed it shouldn't truncate the start).